options/locale/locale_en-US.ini

```
@@ -1742,6 +1745,16 @@ auths.oauth2_authURL = Authorize URL
 auths.oauth2_profileURL = Profile URL
 auths.oauth2_emailURL = Email URL
 auths.enable_auto_register = Enable Auto Registration
+auths.sspi_auto_create_users = Automatically create users
+auths.sspi_auto_create_users_helper = Allow SSPI auth method to automatically create new users on the go (default: true)
```

No need for `(default: ...)` text in helpers
```suggestion
auths.sspi_auto_create_users_helper = Allow SSPI auth method to automatically create new users on the go
```